src/mrtjp/projectred/core/RenderHalo.scala

```diff
@@ -45,7 +45,27 @@ object RenderHalo
         renderList = renderList.sorted
 
         GL11.glPushMatrix()
-        RenderUtils.translateToWorldCoords(Minecraft.getMinecraft.renderViewEntity, event.partialTicks)
+
+        // Removed: To minimize floating point errors we need to minimize calculations (especially 
```

Comment:
Haven't seen too many comments while getting aquainted with the code. If the project's style mandates it, yes - otherwise i prefer this style because i often like to comment out large chunks of code. Multi-line comments tend to make that quite tedious. I'd personally get rid of most of the comment lines and keep them brief - but this is my first contribution which is more or less a hack for a particular problem. I just wanted to make sure ppl know what I'm trying to achieve ;-)